java/HealthVault Mobile/HVSample/src/main/res/values/dimens.xml

```diff
@@ -4,81 +4,121 @@
 	<dimen name="activity_horizontal_margin">16dp</dimen>
 	<dimen name="activity_vertical_margin">16dp</dimen>
 
-
-	<!--Home Page -->
-
 	<!--Weight Page -->
-	<dimen name="marginleft_20dp">20dp</dimen>
-	<dimen name="marginright_20dp">20dp</dimen>
-	<dimen name="margintop_10dp">10dp</dimen>
-	<dimen name="margintop_20dp">20dp</dimen>
-	<dimen name="marginbottom_10dp">10dp</dimen>
-
-	<dimen name="weightlayout_marginleft">20dp</dimen>
-	<dimen name="weightlayout_marginright">200dp</dimen>
-	<dimen name="weightlayout_height_40dp">40dp</dimen>
-	<dimen name="weightlayout_height_20dp">20dp</dimen>
-
-	<dimen name="weight_graphcontainer_height">237dp</dimen>
-	<dimen name="weight_graphcontainer_width">360dp</dimen>
-
-	<dimen name="lastweightlabel_width">75dp</dimen>
-	<dimen name="lastweightlabel_height">20dp</dimen>
-	<dimen name="lastweightlabel_marginleft">20dp</dimen>
-	<dimen name="lastweightlabel_margintop">26dp</dimen>
-
-	<dimen name="lastweighttext_width">80dp</dimen>
-	<dimen name="lastweighttext_height">62dp</dimen>
-	<dimen name="lastweighttext_marginleft">20dp</dimen>
-	<dimen name="lastweighttext_marginright">20dp</dimen>
-	<dimen name="lastweighttext_marginbottom">20dp</dimen>
-
-	<dimen name="lastweightmetric_width">16dp</dimen>
-	<dimen name="lastweightmetric_height">16dp</dimen>
+	<dimen name="horizontal_margin_medium">20dp</dimen>
```

Comment:
I will fix as you suggested.  But I was mainly following the samples from our archived Android project here:  [[[https://microsofthealth.visualstudio.com/Health/_git/archive-health-apps?path=%2FSource%2FAndroid%2FHealthApp%2Fkapp%2Fres%2Fvalues%2Fdimens.xml](https://microsofthealth.visualstudio.com/Health/_git/archive-health-apps?path=%2FSource%2FAndroid%2FHealthApp%2Fkapp%2Fres%2Fvalues%2Fdimens.xml)]([https://microsofthealth.visualstudio.com/Health/_git/archive-health-apps?path=%2FSource%2FAndroid%2FHealthApp%2Fkapp%2Fres%2Fvalues%2Fdimens.xml](https://microsofthealth.visualstudio.com/Health/_git/archive-health-apps?path=%2FSource%2FAndroid%2FHealthApp%2Fkapp%2Fres%2Fvalues%2Fdimens.xml))]



---
In reply to: [120484996](https://github.com/Microsoft/healthvault-samples/pull/13#discussion_r120484996) [](ancestors = 120484996)